src/main/java/com/taf/business/pages/LoginPage.java

```diff
@@ -0,0 +1,19 @@
+package com.taf.business.pages;
+
+import com.taf.core.ConfigManager;
+
+import static com.codeborne.selenide.Condition.enabled;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class LoginPage {
+    private static final String LOGIN_FIELD = "//input[@name='login']";
+    private static final String PASSWORD_FIELD = "//input[@name='password']";
+    private static final String SUBMIT_BUTTON = "//button[@type='submit']";
+
+    public void fillLoginForm() {
```

Comment:
It would be better to avoid business steps on the page layer and leave only actions with the page here. For example, you can leave three methods like fillLoginField, fillPasswordField, and clickSubmitButton here and make one more layer between Pages and Tests with more complex logic and call it like Service layer or Business flow layer.

Reply:
It is a good idea. Done.